example/lib/custom_code_box.dart

```diff
@@ -55,16 +56,20 @@ class _CustomCodeBoxState extends State<CustomCodeBox> {
   ];
 
   List<String?> themeList = <String>[
-    "monokai-sublime",
-    "a11y-dark",
-    "an-old-hope",
-    "vs2015",
-    "vs",
-    "atom-one-dark",
+    'monokai-sublime',
+    'a11y-dark',
+    'an-old-hope',
+    'vs2015',
+    'vs',
+    'atom-one-dark',
   ];
 
-  Widget buildDropdown(Iterable<String?> choices, String value, IconData icon,
-      Function(String?) onChanged) {
+  Widget buildDropdown(
```

Comment:
Why aren't reset, languageList, buildDropdown, etc. private?

Reply:
I only fixed linter issues. This will be done later.